src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
+
+	browser.pause(2000);
+
+	const cookieControl = $("body #ccc");
+		if (cookieControl.isVisible("button=Accept all cookies")) {
+		cookieControl.$("button=Accept all cookies").click();
+	}
+		else (browser.refresh())
```

Comment:
I'm not sure you should need a refresh - if the accept button isn't visible then you don't need to do anything

Reply:
that seem to work now with the waitFor global-nav now in place.